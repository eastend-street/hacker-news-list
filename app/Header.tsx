export default function Header() {
  return (
    <header className="flex items-center justify-center bg-orange py-1">
      <h1 className="text-1xl font-bold">Hacker News List</h1>
    </header>
  );
}
