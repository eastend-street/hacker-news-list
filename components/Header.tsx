export default function Header() {
  return (
    <header className="flex items-center bg-orange py-1 px-3">
      <h1 className="text-1xl font-bold">Hacker News List</h1>
    </header>
  );
}
