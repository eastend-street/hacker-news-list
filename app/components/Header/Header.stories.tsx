import React from 'react';

import Header from './Header';

export default {
  title: 'components/Header',
  component: Header,
};

export const Template = () => <Header />;

export const Default = Template.bind({});
