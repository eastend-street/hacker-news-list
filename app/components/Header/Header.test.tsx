import { render, screen } from '@testing-library/react';
import { composeStories } from '@storybook/react';

import * as stories from './Header.stories';

const { Default: Header } = composeStories(stories);

describe('components/Header', () => {
  test('should render Header component', () => {
    const { container } = render(<Header />);

    const heading1 = screen.getByRole('heading', {
      name: /Hacker News List/i,
      level: 1,
    });

    expect(heading1).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
