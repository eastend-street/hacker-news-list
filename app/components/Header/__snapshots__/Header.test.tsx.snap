// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header should render Header component 1`] = `
<div>
  <header
    class="flex items-center bg-orange py-1 px-3"
  >
    <h1
      class="text-1xl font-bold"
    >
      Hacker News List
    </h1>
  </header>
</div>
`;
